### ISSUE TITLE
Installation times out

### ISSUE THREAD

**Issue by stefanbschneider** [RealVNF/distributed-drl-coordination#3]
Hi, When I run `pip install -r requirements.txt`,  it always times out:
```
 error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet git://github.com/RealVNF/common-utils /home/guohao/PycharmProjects/codes/venv/src/common-utils did not run successfully.
  │ exit code: 128
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet git://github.com/RealVNF/common-utils /home/guohao/PycharmProjects/codes/venv/src/common-utils did not run successfully.
│ exit code: 128
╰─> See above for output.
```
How can I solve this problem? thank you

_Originally posted by @BMDACMER in https://github.com/RealVNF/distributed-drl-coordination/issues/2#issuecomment-1521752202_

**Comment by stefanbschneider** [comment]
I'll first try to properly release `coord-sim` so it becomes more usable and we can simplify this setup: https://github.com/RealVNF/coord-sim/issues/210

**Comment by stefanbschneider** [comment]
I made some fixes and the setup should work once #4 is merged. Feel free to reopen the issue if it doesn't work.
